README.md

```diff
@@ -3,7 +3,7 @@ must-gather
 
 `openshift-must-gather` is a tool for collecting cluster data.
 It dumps `clusteroperator` data, and associated namespace data, into a specified `--base-dir` location.
-The directory structure, as well as specific details behind this tool can be found [in this enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/must-gather.md).
+The directory structure, as well as specific details behind this tool can be found [in this enhancement](https://github.com/openshift/must-gather/blob/master/must-gather.md).
```

Comment:
I think we want the broken link to pint to https://www.github.com/openshift/enhancements/tree/master/enhancements%2Foc%2Fmust-gather.md not the document in this repo.

Reply:
Thanks, fixed the commit to point to correct document.